docs/ui-principles.rst

```diff
@@ -61,7 +61,7 @@ help content).
 -------------------------------------------------------
 
 Warehouse follows the `Material design writing style guide
-<https://material.io/design/usability/accessibility.html#writing>`_.
+<https://web.archive.org/web/20180410101124/https://material.io/guidelines/style/writing.html>`_.
```

Comment:
Why did this link change? Looks like https://material.io/design/usability/accessibility.html#writing still exists to me.

Reply:
IIRC, this was the link originally and then this was changed when material design got an update, which broke the links.

Maybe we should follow the latest version of Material on this; but that's a moving target so, choosing an archived version also makes sense.

Reply:
As far as I can tell, they have reduced the guidelines in the newer version, which is why I asked to change to the archived link.  I agreewith @pradyunsg that it makes sense to pin to a particular version.  Another option would be to add this documentation into our own docs, so that we were not affected by future changes, or could make changes to the guidelines as/when we needed.

Reply:
ticket opened: https://github.com/pypa/warehouse/issues/4422